RealTimeFaceMaskDetector/LogViewerTests/test.cpp

```diff
@@ -0,0 +1,110 @@
+#include "pch.h"
+#include "../LogViewer/Parser.cpp"
+#include "../LogViewer/structures.cpp"
+#include "../LogViewer/Filter.cpp"
+#include "../DateTimeDLL/DateTime.cpp"
+#include <list>
+
+const std::string path{ "C:/Users/Oleksii/Documents/Real-Time-Face-Mask-Detector-Server/TestData/testLog.log" };
+void check_logs_by_options(size_t, std::vector<logger::LogOptions>);
+void check_logs_by_time(size_t, const std::string&);
+
+TEST(LogViewerLogs, AllLogs) {
+	constexpr int number_of_logs{ 12 };
+	std::vector<logger::LogOptions> options{
+		logger::LogOptions::INFO,
+		logger::LogOptions::WARNING,
+		logger::LogOptions::DEBUG,
+		logger::LogOptions::ERR,
+		logger::LogOptions::FATAL,
+		logger::LogOptions::TRACE
+	};
+	std::string from{ "0000-00-00 00:00:00" };
+	std::string till{ "0000-00-00 00:00:00" };
+	auto parser = logger::Parser(path);
+	parser.parse(options, from, till);
+	auto logs = parser.get_logs();
+
+	EXPECT_EQ(logs.size(), number_of_logs);
+}
+
+//TEST(LogViewerTests, InfoLogs) {
```

Comment:
Are these comments necessary?

Reply:
I've fixed it, for now we can leave these tests commented